cms/templates/js/mock/mock-manage-users-lib.underscore

```diff
@@ -1,146 +1,57 @@
+      <header class="mast has-actions has-subtitle">
+        <h1 class="page-header">
+          <small class="subtitle">Settings</small>
+          <span class="sr">&gt; </span>Instructor Access
+        </h1>
+
+        <nav class="nav-actions">
+          <h3 class="sr">Page Actions</h3>
+          <ul>
+            <li class="nav-item">
+              <a href="#" class="button new-button create-user-button"><i class="icon fa fa-plus"></i> Add Instructor</a>
             </li>
           </ul>
-
-        </li>
-
-        <li class="user-item" data-email="audit@example.com">
-
-          <span class="wrapper-ui-badge">
-            <span class="flag flag-role flag-role-admin is-hanging">
-              <span class="label sr">Current Role:</span>
-              <span class="value">
-                Admin
-              </span>
-            </span>
-          </span>
-
-          <div class="item-metadata">
-            <h3 class="user-name">
-              <span class="user-username">audit</span>
-              <span class="user-email">
-                <a class="action action-email" href="mailto:audit@example.com" title="send an email message to audit@example.com">audit@example.com</a>
-              </span>
-            </h3>
-          </div>
-
-          <ul class="item-actions user-actions">
-                <li class="action action-role">
-                  <a href="#" class="make-staff admin-role remove-admin-role">Remove Admin Access</span></a>
-                </li>
-            <li class="action action-delete ">
-                <a href="#" class="delete remove-user action-icon" data-tooltip="Remove this user"><i class="icon fa fa-trash-o"></i><span class="sr">Delete the user, audit</span></a>
-            </li>
-          </ul>
-
-        </li>
-
-        <li class="user-item" data-email="staff@example.com">
-
-          <span class="wrapper-ui-badge">
-            <span class="flag flag-role flag-role-user is-hanging">
-              <span class="label sr">Current Role:</span>
-              <span class="value">
-                User
-              </span>
-            </span>
-          </span>
-
-          <div class="item-metadata">
-            <h3 class="user-name">
-              <span class="user-username">staff</span>
-              <span class="user-email">
-                <a class="action action-email" href="mailto:staff@example.com" title="send an email message to staff@example.com">staff@example.com</a>
-              </span>
-            </h3>
+        </nav>
+      </header>
+    </div>
+
+    <div class="wrapper-content wrapper">
+      <section class="content">
+        <article class="content-primary" role="main">
+          <div class="wrapper-create-element animate wrapper-create-user">
+            <form class="form-create create-user" id="create-user-form" name="create-user-form">
+              <div class="wrapper-form">
+                <h3 class="title">Grant Instructor Access to This Library</h3>
```

Comment:
It would have been nice if this "add user" section of the page was also in Backbone. I won't hold up the PR for it, but did you consider it?